tools/vigiechiro/.shed.yml

```diff
@@ -36,25 +36,23 @@ repositories:
       - test-data/BilanEnrichiRP_Test1_out_tab.tabular
       - test-data/BilanEnrichiRP_Test1.tabular 
 
-  vigiechiro_idcorrect_2ndlayer: 
+  vigiechiro_idtidy: 
```

Comment:
What happened to the old tools?

Reply:
hmm, maybe a mistake as if I well understood, idtidy is not a new version of 2ndlayer but a new tool isn't it ?

Reply:
The first input table come from the vigiechiro website, they integrated in it a part of idcorrect2ndlayer so the classifier is not needed anymore.
The files in the workflow are slightly different so the tools.
Tidy is the new name, I don't really know what it is about, but 'idcorrect_2ndlayer' doesn't match the functionality now.